statsprinter.go

```diff
@@ -26,10 +36,54 @@ var (
 	colorLightCyan   = color.LightCyan.Printf
 )
 
-type planePrinter struct{}
+type planePrinter struct {
+	ShowTimestamp *bool
+}
+
+type PrintOptions struct {
+	Color         Color
+	Message       string
+	ShowTimestamp *bool
+}
```

Comment:
why do you need to export this struct and its fields?

Reply:
We don't need to, This is my mistake...I've fixed it.